src/App.js

```diff
@@ -1,19 +1,29 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 import {Navbar} from './components/Navbar.js'
 import { AddFacility } from './components/AddFacility.js';
 
 
 import './App.css';
-import { GridContainer } from './components/CardsContainer.js';
+import { CardsContainer} from './components/CardsContainer.js';
+import facilitiesData from './facilities.json'
 
 
 function App() {
+
+  localStorage.setItem('facilities', {facilitiesData})
```

Comment:
make a separate service that handles communication with local storage

Reply:
fixed